src/components/ButtonSubmit.js

```diff
@@ -0,0 +1,59 @@
+import { useState } from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+
+function Underline(props) {
+  return (
+    <svg viewBox="0 0 141 9" fill="none" {...props}>
+      <path
+        d="M2.41256 6.22467C19.7092 4.09854 71.0272 0.832086 137.927 4.7753"
+        stroke="#EFBF4D"
+        strokeWidth="4.5301"
```

Comment:
The stroke is a bit thick for this size button. Setting this value to 3 fixes that.